infrastructure/core/network.tf

```diff
@@ -77,3 +77,46 @@ resource "azurerm_private_dns_zone_virtual_network_link" "all" {
     azurerm_private_dns_zone.all
   ]
 }
+
+resource "azurerm_network_security_group" "core" {
```

Comment:
Did databricks deploy ok in your testing? Nothing jumps out that would be blocked that databricks needs but worth us being sure

Reply:
let's wait and see 😅 https://github.com/UCLH-Foundry/FlowEHR/actions/runs/4552205929/jobs/8027474893

Reply:
databricks creates its own NSG with a bunch of 100 ish priorities so as long as NSGs get merged we should(🤞🏼) be ok

Reply:
`make test-transform` works locally